composer.json

```diff
@@ -23,19 +23,19 @@
         }
     },
     "require": {
-        "php": ">=7",
-        "amphp/parallel": "^1.1",
-        "amphp/amp": "^2.0.3",
-        "opis/closure": "^3.0.7"
+        "php": ">=7.4",
+        "amphp/parallel": "^1.4.1",
+        "amphp/amp": "^2.6.1",
+        "laravel/serializable-closure": "^1.0"
     },
     "require-dev": {
-        "amphp/phpunit-util": "^1.0",
-        "friendsofphp/php-cs-fixer": "^2.9",
-        "phpunit/phpunit": "^6.5"
+        "amphp/phpunit-util": "^2.0",
+        "friendsofphp/php-cs-fixer": "^3.5",
```

Comment:
maybe the PHP-CS-Fixer upgrade can be moved into a separate PR?

Reply:
Moved to https://github.com/amphp/parallel-functions/pull/30 👍🏻